app/js/controllers/foyer/pensionsAlimentaires.js

```diff
@@ -51,31 +49,29 @@ angular.module('ddsApp').controller('FoyerPensionsAlimentairesCtrl', function($s
         $scope.individusVM.push(createIndividuVM(conjoint));
     }
 
-    $scope.hasPensionsAlimentaires = false;
+    $scope.situation.hasPensionsAlimentaires = false;
```

Comment:
Usually you pay for child support. Changing it to `situation.parentsPayPensionsAlimentaires`